cmake/module/FlagsSummary.cmake

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2024-present The Bitcoin Core developers
```

Comment:
nit: here we have the current year, would it make sense to change all of them to 2024 as a slightly better documentation of historical changes?

Reply:
If you mean the "2023" instances in the copyright headers, they reflect the actual time when the code was written.